src/lib/components/Content.svelte

```diff
@@ -6,13 +6,18 @@
   export let title: string = '';
   export let subtitle: string = '';
   export let backUrl: string = '';
+  export let backAction: () => void = () => {};
   export let backTitle: string = 'Back';
   export let full = false;
+
+  function onBack() {
+    backAction();
+    goto(backUrl);
+  }
 </script>
 
 <section class={`main-content ${full ? 'w-full' : ''}`}>
```

Comment:
Is it okay if I make a component for this using app bar? I need to use it as well.